plugin.rb

```diff
@@ -938,10 +939,11 @@ def show
 
     def create
       # guardian.ensure_can_create_explorer_query!
-
+      
       query = DataExplorer::Query.from_hash params.require(:query)
-      query.created_at = Time.now.strftime("%b %e, %Y")
+      query.created_at = Time.now
       query.created_by = current_user.id.to_s
+      query.last_run_at = '-'
```

Comment:
I think we should default `last_run_at` to the time query was created instead of keeping it blank.